docs/README.rst

```diff
@@ -516,48 +535,51 @@ You can use the |kctlr| to `Expose Services to External Traffic using Ingresses`
++----+------------------------------------------+-------------+-----------+-------------------------------------------------------------------------------------+-------------+-----------------------------------------+
+|    | interval                                 | integer     | Required  | The interval at which to check the health of the virtual server.                    | N/A         |                                         |
+|    |                                          |             | [#hm1]_   |                                                                                     |             |                                         |
++----+------------------------------------------+-------------+-----------+-------------------------------------------------------------------------------------+-------------+-----------------------------------------+
+|    | timeout                                  | integer     | Required  | Number of seconds before the check times out.                                       | N/A         |                                         |
+|    |                                          |             | [#hm1]_   |                                                                                     |             |                                         |
++----+------------------------------------------+-------------+-----------+-------------------------------------------------------------------------------------+-------------+-----------------------------------------+
+|    | send                                     | string      | Required  | The send string to set in the health monitor. [#hm2]_                               | N/A         |                                         |
+|    |                                          |             | [#hm1]_   |                                                                                     |             |                                         |
++----+------------------------------------------+-------------+-----------+-------------------------------------------------------------------------------------+-------------+-----------------------------------------+
+|    | recv                                     | string      | Optional  | String or RegEx pattern to match in first 5,120 bytes of backend response.          | N/A         |                                         |
++----+------------------------------------------+-------------+-----------+-------------------------------------------------------------------------------------+-------------+-----------------------------------------+
+| virtual-server.f5.com/source-addr-translation | string      | Optional  | Source address translation type to apply to the virtual server.                     | automap     | automap, none, snat                     |
```

Comment:
how is the snat pool specified (same comment for routes too)?

Reply:
The full annotation would be: `"virtual-server.f5.com/source-addr-translation": "{"type":"snat", "pool":"snat-pool-name"}"`